src/Responses/SuccessResponseGenerator.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Mtrajano\LaravelSwagger\Responses;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Str;
+use Mtrajano\LaravelSwagger\DataObjects\Route;
+use ReflectionException;
+
+class SuccessResponseGenerator
+{
+    /**
+     * @var Route
+     */
+    private $route;
+    /**
+     * @var Model
+     */
+    private $model;
+
+    public function __construct(Route $route)
+    {
+        $this->route = $route;
+    }
+
+    /**
+     * @return array
+     * @throws ReflectionException
+     */
+    public function generate()
+    {
+        $methodMappingHttpCode = [
+            'get' => '200',
+            'post' => '201',
+            'put' => '204',
+            'delete' => '204',
+        ];
+
+        // Get the status code from route method
+        $methods = $this->route->validMethods();
+
+        // TODO: Handle with many methods in same route. E.g.: Route::match(['GET', 'POST']);
+        $httpCode = $methodMappingHttpCode[$methods[0]];
+
+        $description = $this->getDescriptionByHttpCode($httpCode);
+
+        $this->setModelFromRouteAction();
+        if (!$this->model) {
+            return [];
+        }
+
+        $response = [
+            $httpCode => [
+                'description' => $description,
+            ]
+        ];
+
+        $schema = $this->mountSchema($httpCode);
+        if (!empty($schema)) {
+            $response[$httpCode]['schema'] = $schema;
+        }
+
+        return $response;
+    }
+
+    private function getDescriptionByHttpCode(string $httpCode)
+    {
+        $httpCodeDescription = [
+            '200' => 'OK',
+            '201' => 'Created',
+            '204' => 'No Content',
+        ];
```

Comment:
could also be moved to the config, that way the user can choose whichever description he wants and add more if needed